x-pack/test_serverless/api_integration/test_suites/common/platform_security/authorization.ts

```diff
@@ -8,8 +8,20 @@
 import expect from 'expect';
 import { KibanaFeatureConfig, SubFeaturePrivilegeConfig } from '@kbn/features-plugin/common';
 import { SupertestWithRoleScopeType } from '@kbn/test-suites-xpack/api_integration/deployment_agnostic/services';
+import type { Role } from '@kbn/security-plugin-types-common';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
+/*
+ * This file contains the common authorization tests that are applicable to
+ * all peroject types. There are two other authorization test files due to
```

Comment:
__question:__ hmm, I didn't manage to find these files (`authorization_custom_roles.ts` and `authorization_oblt.ts`) 🙈 Where are they?